src/main/java/org/polystat/Program.java

```diff
@@ -68,8 +68,11 @@ public Program(final Path src, final Path tmp) {
     public XML apply(final String locator) throws Exception {
         final String[] parts = locator.split("\\.");
         final String name = parts[1];
-        final Path xml = this.temp.resolve(String.format("%s.xml", name));
-        final Path src = this.sources.resolve(String.format("%s.eo", name));
+        final Path xml = this.temp.resolve(String.format("%s.xml", name)).toAbsolutePath();
```

Comment:
@pollyvolk what caused the path to be changed to absolute? Were there issues with previous version of the code?